.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+*Please describe what changes you made, and why you feel they are necessary. Make sure to include
+code examples, where applicable.*
+
+Corresponding issue:
+
+## Testing
+*Please explain how you tested this change manually, and, if applicable, what new tests you added. If
+you're making a change to just the website, you can omit this section.*
```

Comment:
Is this section really necessary? What do you think people have to write here?

Reply:
I get your point, but I want people to describe their testing strategy, partially as a way to verify that they have one (and encourage them to develop one, if they don't already)